configure

```diff
@@ -567,6 +567,7 @@ check check_pthread
 check check_rtsched
 check check_ncurses
 check check_iconv
+check check_libintl
```

Comment:
I'd rather also have something like `check_header libintl.h && check_libintl && HAVE_LIBINTL=y` here, as this will produce better feedback if libintl is missing.